docs/index.md

```diff
@@ -11,7 +11,7 @@
 
 A [Pulp](https://pulpproject.org) plugin to support hosting your very own Ansible Galaxy server.
 
-This is a brand new take on Ansible Galaxy, so it will look and feel a bit different than the current [galaxy.ansible.com web site](https://galaxy.ansible.com). Over time we expect to migrate the web site to this codebase, so for now you're looking into the future, and you have an opportunity to help shape that future.
+This is a brand new take on Ansible Galaxy, so it will look and feel a bit different than the old [galaxy.ansible.com web site](https://old-galaxy.ansible.com).
```

Comment:
HI, I think we can omit this statement, no need to mention old-galaxy, which is still out there only for backup reasons.

Reply:
Also true!
Or keep it somewhere as a link in case people want to refer old galaxy (as the new galaxy-ng still missing some of the features!)